scripts/vqvae_test.py

```diff
@@ -0,0 +1,78 @@
+import logging
+from pathlib import Path
+
+import torch
+from huggingface_hub import hf_hub_download
+from PIL import Image
+from torchvision import transforms as T
+from torchvision.utils import save_image
+
+from muse_maskgit_pytorch.vqvae import VQVAE
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+# where to find the model and the test images
+model_repo = "neggles/vaedump"
+model_subdir = "vit-s-vqgan-f4"
+test_images = ["testimg_1.png", "testimg_2.png"]
+
+# where to save the preprocessed and reconstructed images
+image_dir = Path.cwd().joinpath("temp")
+image_dir.mkdir(exist_ok=True, parents=True)
+
+# image transforms for the VQVAE
+transform_enc = T.Compose([T.Resize(512), T.RandomCrop(256), T.ToTensor()])
+transform_dec = T.Compose([T.ConvertImageDtype(torch.uint8), T.ToPILImage()])
+
+
+def get_save_path(path: Path, append: str) -> Path:
+    return path.with_stem(f"{path.stem}-{append}")
+
+
+def main():
+    torch_device = torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu")
+    dtype = torch.float32
+
+    # load VAE
+    logger.info(f"Loading VQVAE from {model_repo}/{model_subdir}...")
+    vae: VQVAE = VQVAE.from_pretrained(model_repo, subfolder=model_subdir, torch_dtype=dtype)
+    vae = vae.to(torch_device)
+    logger.info(f"Loaded VQVAE from {model_repo} to {vae.device} with dtype {vae.dtype}")
+
+    # download and process images
+    for image in test_images:
+        image_path = hf_hub_download(model_repo, subfolder="images", filename=image, output_dir=image_dir)
```

Comment:
On the latest version of the hf_hub the `output_dir` is gone, we should use `local_dir` instead.

Reply:
Ah yep. I blame GitHub Copilot 😅 will fix